releasenotes/notes/0.9/new-features-0.9-159645f977a139f7.yaml

```
@@ -0,0 +1,205 @@
+---
+prelude: >
+    The 0.9 release includes many new features and many bug fixes. The biggest
+    changes for this release are new debugging capabilities for PassManagers.
+    This includes a function to visualize a PassManager, the ability to add
+    a callback function to a PassManager, and logging of passes run in the
+    PassManager. Additionally, this release standardizes the way that
+    you can set an initial layout for your circuit. So now you can leverage
+    ``initial_layout`` the kwarg parameter on ``qiskit.compiler.transpile()``
+    and ``qiskit.execute()`` and the qubits in the circuit will get laid
+    out on the desire qubits on the device. Visualization of circuits will
+    now also show this clearly when visualizing a circuit that has been
+    transpiled with a layout.
+features:
+  - |
+    A new analysis pass ``CountOpsLongest`` was added to retrieve the number
+    of operations on the longest path of the DAGCircuit. When used it will
+    add a ``count_ops_longest_path`` key to the property set dictionary.
+    You can add it to your a passmanager with something like::
+
+        from qiskit.transpiler.passes import CountOpsLongestPath
+        from qiskit.transpiler.passes import CxCancellation
+        from qiskit.transpiler import PassManager
+
+        pm = PassManager()
+        pm.append(CountOpsLongestPath)
```

This is incorrect. It should be
```suggestion
        pm.append(CountOpsLongestPath())
```
Shall I change the file directly?